xbmc/utils/URIUtils.cpp

```diff
@@ -88,35 +88,80 @@ bool URIUtils::HasExtension(const std::string& strFileName)
   return iPeriod != std::string::npos && strFileName[iPeriod] == '.';
 }
 
-bool URIUtils::HasExtension(const CURL& url, const std::string& strExtensions)
+bool URIUtils::HasExtension(const CURL& url, std::string_view strExtensions)
```

Comment:
Could you please remove the type prefix from the string param?

=> https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#87-methods